libpam/include/security/_pam_macros.h

```diff
@@ -94,11 +94,31 @@ do {                                              \
 #define _PAM_LOGFILE "/var/run/pam-debug.log"
 #endif
 
+#ifdef PAM_NO_HEADER_FUNCTIONS
+#ifdef PAM_DEBUG_C
+#define PAM_DEBUG_TYPE void
+#else
+#define PAM_DEBUG_TYPE extern void
+#endif
```

Comment:
I think we can just define `PAM_DEBUG_TYPE` unconditionally here because `void` and `extern void` mean the same in declarations.

Reply:
In that case we can drop the whole PAM_DEBUG_C aspect here.